README.md

```diff
@@ -0,0 +1,105 @@
+# Scylla Modern C++ Driver
+This project (Scylla Modern C++ Driver, later referred to as SCMD) aims to provide easy way to use Scylla from C++. DataStax cpp driver only exports C API, which isn't convenient to use in C++. It does use C++ internally, but unfortunately this internal API is not intended to be used by library users. SCMD is a small library that wraps this C API to provide classes, RAII, templates (instead of functions with type in name) etc.
+
+## Building
+Put this repo in a folder in your project (e.g. as a git submodule), and in your CMakeLists.txt add line:
+```
+add_subdirectory(path/to/driver EXCLUDE_FROM_ALL)
+```
+Then you need to link library to targets that use it:
+```
+target_link_libraries(your_library scylla_modern_cpp_driver)
+```
+
+That's it, now you can just `#include "scmd.hh"` and use library.
+
+## Testing
+Tests are built when SCMD is built as a standalone project, not a subdirectory. To use them, you need to have a working scylla running - there is a script `run_scylla.sh` in the repo to run scylla in docker. To run tests execute  `./build_directory/tests/tests -- scylla_ip`.
+
+## Usage
+```C++
+//====== Connecting =======
+// Constructing scmd::session objects makes new connection to Scylla.
+// Pass IP and optionally port of Scylla instance.
+// This API is most likely going to change
+scmd::session session("172.19.0.2", "4092");
+
+//====== Basic queries =======
+// Simple query, no parameters
+scmd::statement stmt("SELECT * FROM test");
+
+// Query with parameters. You need to pass number of parameters to constructor
+scmd::statement stmt2("SELECT * FROM test WHERE param1 = ? AND param2 = ?", 2);
+// It's important to pass variables with right types to the bind method, otherwise query will fail.
+stmt2.bind(var1, var2);
```

Comment:
This makes a great test case. Also, what happens when it fails can be included in a test case.

Reply:
"This makes a great test case" - What makes a great test case? Current test cases already use binding, with many different types. 

"Also, what happens when it fails can be included in a test case." - done